### ISSUE TITLE
Configuration changes do not save

### ISSUE THREAD

**Issue by nicksilver** [davidrthorn/cross_reference#14]
Hi, I cannot seem to get my configuration changes to be implemented or saved. Any suggestions? I am wondering if there is some sort of conflict with style formatting.

**Comment by davidrthorn** [comment]
Hi Nick

Just to clarify, do you mean you can't change the text/formatting for existing reference types or you can't add custom types?

Regarding the first, I've been able to recreate this to some extent, but haven't tracked down the problem. One thing that worked for me was closing the sidebar and reopening it to change the settings. Does this work for you?

**Comment by nicksilver** [comment]
Hi David,

As you suggested, I was referring to the former. I closed the sidebar
several times and reopened. Nothing helped. Then I went to work and got on
a different computer (same OS and browser) and everything seems to be
working fine. I cannot reproduce the issue now so I have no idea what the
problem was. I am thinking maybe I should have closed down Docs and
reopened to see if that did the trick on the other computer.

Now that things are working, I can see that there are options in the
sidebar that were not visible to me when I was on my other computer. The
main ones being the drop down menu on the top that allows a user to select
the type of reference and label (i.e. equation, figure, or table). This
dropdown was not functioning on the other computer. Nor were there any
default text in the "Text" boxes...those were empty.

For now I am good...and thanks for reaching out...and thanks for creating
this very useful add-on. Much appreciated!

Nick
